File: debug_function.js

const url = "https://jzylycxvjmxzyfpyhngx.supabase.co/functions/v1/create-checkout";
const headers = {
    "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imp6eWx5Y3h2am14enlmcHlobmd4Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjcyOTExMjksImV4cCI6MjA4Mjg2NzEyOX0.3gjVuMMX0YgfP3KhR5DxLAWe9iwzKiZ4BhJdgh8vb6o",
    "Content-Type": "application/json"
};

async function test() {
    try {
        console.log(`Testing URL: ${url}`);
        const response = await fetch(url, {
            method: 'POST',
            headers: headers,
            body: JSON.stringify({})
        });

        console.log(`Status Code: ${response.status}`);
        const text = await response.text();
        console.log(`Response Body: ${text}`);
    } catch (error) {
        console.error(`Error: ${error.message}`);
    }
}

test();

File: debug_function.py
import requests
import json

url = "https://jzylycxvjmxzyfpyhngx.supabase.co/functions/v1/create-checkout"
headers = {
    "Authorization": "Bearer REMOVED_FOR_SAFETY", 
    "Content-Type": "application/json"
}

try:
    print(f"Testing URL: {url}")
    response = requests.post(url, headers=headers, json={}, timeout=10)
    print(f"Status Code: {response.status_code}")
    print(f"Response Body: {response.text}")
except Exception as e:
    print(f"Error: {e}")
